#!/usr/bin/env python
# Arips Planner configuration

from dynamic_reconfigure.parameter_generator_catkin import ParameterGenerator, double_t, int_t, bool_t

gen = ParameterGenerator()

gen.add("controller_patience", double_t, 0, "Time for local controller to wait when controller fails to produce velocity commands before running recovery behavior", 1.0, 0.0, 10.0)
gen.add("fault_patience", double_t, 0, "Time for locomotion to try alternatives until a total fail is considered", 10.0, 0.0, 100.0)
gen.add("replan_every_step", bool_t, 0, "Do path planning to goal on every control cycle", True)

exit(gen.generate("arips_navigation", "arips_navigation", "Locomotion"))
